pkg/protoc/wrapper.go

```diff
@@ -0,0 +1,118 @@
+package protoc
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+
+	"github.com/izumin5210/clicontrib/pkg/clog"
+	"github.com/izumin5210/gex/pkg/tool"
+	"github.com/pkg/errors"
+	"github.com/spf13/afero"
+	"k8s.io/utils/exec"
+
+	"github.com/izumin5210/grapi/pkg/clui"
+	"github.com/izumin5210/grapi/pkg/grapicmd/util/fs"
+)
+
+type Wrapper interface {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type Wrapper should have comment or be unexported

Reply:
fixed in d26e797